### PR TITLE
feat(core): adds angular 15 support

### PR DESCRIPTION
**Pull request by nikes**

### CODE REVIEW

**Comment by doughlass** [comment]
@Napster2210 can we get this merged and released?

**Comment by doughlass** [comment]
image

**Comment by Napster2210** [comment]
Hello guys, sorry for the delay but I am out of the station and will be back by early next week. I will surely release by then.

**Review by Rebstorm — APPROVED** [review]

File: projects/ngx-tester/src/app/app.module.ts [hunk 1]
```diff
@@ -2,24 +2,21 @@ import { BrowserModule } from '@angular/platform-browser';
...
-import { MatButtonModule } from '@angular/material/button';
-import { MatSelectModule } from '@angular/material/select';
 import { MatExpansionModule } from '@angular/material/expansion';
-import { MatSlideToggleModule } from '@angular/material/slide-toggle';
-
-import { NgxSpinnerModule } from 'ngx-spinner';
 import { AppComponent } from './app.component';
 import { HeaderComponent } from './header/header.component';
 import { BannerComponent } from './banner/banner.component';
+import {MatFormFieldModule} from '@angular/material/form-field';
```
[the remark this answers is not in the document]
Comment (doughlass) [reply projects/ngx-tester/src/app/app.module.ts:13]: tbh these should be auto formatted upon save if they are mandatory

**Comment by doughlass** [comment]
@nikes are you able to lint your branch?

**Comment by alexvrv** [comment]
Any news?...

**Pull request closed without merging**